core/testing/src/main/java/io/github/droidkaigi/confsched2023/testing/robot/TimetableScreenRobot.kt

```diff
@@ -50,12 +53,14 @@ class TimetableScreenRobot @Inject constructor(
 
     fun setupTimetableScreenContent() {
         composeTestRule.setContent {
-            KaigiTheme {
-                TimetableScreen(
-                    onSearchClick = { },
-                    onTimetableItemClick = { },
-                    onBookmarkIconClick = { },
-                )
+            CompositionLocalProvider(LocalClock provides FakeClock) {
```

Comment:
@takahirom 
It does not seem like it works well...
How can I overwrite the CompositionLocal in test environment? 🤔

Maybe, since there is no setContent in KaigiAppTest, so the only way is to add like ClockTestRule or something like that?

Reply:
I'm not sure too. 
By the way, you might have to fix the commit history 👀

Reply:
Thank you, I'll investigate the cause again 🙏
And regarding your BTW comment, which commits? All?

Reply:
Sorry, I might see old commit 🙇

Reply:
The FakeClock worked well for only TimetableScreenTest on my local, but the screenshot results haven't been updated even if I added some commits 👀
Is there any way to update them on this PR's comment?

In case of `2023-09-14T10:00:00.000Z`
![TimetableScreenTest checkGridScrollShot_compare](https://github.com/DroidKaigi/conference-app-2023/assets/43767445/76999b05-6610-423f-8688-06f87556635e)

In case of `2023-09-15T10:00:00.000Z` (To check the diff)
![TimetableScreenTest checkGridScrollShot_compare](https://github.com/DroidKaigi/conference-app-2023/assets/43767445/87b00b8c-37fc-4525-9c51-6b6addd84bce)

Reply:
I've removed the comment. Let's see what screenshots the tests take.

Reply:
<img width="1230" alt="image" src="https://github.com/DroidKaigi/conference-app-2023/assets/1386930/71f34998-b16c-4c83-a1aa-d50235465a8b">
It seems that there is no change 🎉

Reply:
Thank you! 🙌